pkg/diagnostics/tracing.go

```diff
@@ -30,18 +35,17 @@ const (
 
 	// span attribute keys
 	// Reference trace semantics https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/trace/semantic_conventions
-	dbTypeSpanAttributeKey      = "db.type"
-	dbInstanceSpanAttributeKey  = "db.instance"
-	dbStatementSpanAttributeKey = "db.statement"
-	dbURLSpanAttributeKey       = "db.url"
+	dbTypeSpanAttributeKey      = string(semconv.DBSystemKey)
```

Comment:
Can you rename `dbTypeSpanAttributeKey` to `dbSystemSpanAttributeKey` ? please rename the other const as well..

Reply:
Done